test_sqlalchemy.py

```diff
@@ -456,6 +456,44 @@ class Baz(db.Model):
             Baz.__table__: db.get_engine(app, None)
         })
 
+    def test_abstract_binds(self):
+        _, db1 = tempfile.mkstemp()
+        _, db2 = tempfile.mkstemp()
+
+        def _remove_files():
+            try:
+                os.remove(db1)
+                os.remove(db2)
+            except IOError:
+                pass
+        atexit.register(_remove_files)
```

Comment:
There's no need for creating and removing db files, just use `sqlite://` for all binds, each will connect to a different in-memory db.